qtfred/src/ui/dialogs/ShipEditor/ShipInitialStatusDialog.h

```diff
@@ -19,18 +18,40 @@ class ShipInitialStatusDialog : public QDialog {
 	explicit ShipInitialStatusDialog(QDialog* parent, EditorViewport* viewport, bool editMultiple);
 	~ShipInitialStatusDialog() override;
 
+	void accept() override;
+	void reject() override;
+
   protected:
 	void closeEvent(QCloseEvent*) override;
-	void rejectHandler();
 
   private slots:
+	void on_okPushButton_clicked();
+	void on_cancelPushButton_clicked();
```

Comment:
Same thing about the QDialogButtonBox as above.

Reply:
Not doing for this one as it would break grid layout.